Lesson 5

rex1k opened the pull request:
rwer

rex1k committed:
er

rwer

rex1k committed:
Lol

rex1k commented on lesson03/home_work/hw03_normal.py:
```diff
@@ -12,16 +27,85 @@ def fibonacci(n, m):
 
 
 def sort_to_max(origin_list):
-    pass
```
Forget to delete this pass

rex1k committed:
Надо уже поспать...

rex1k committed:
апвп

Лессон 4

manchenkoff commented on lesson04/home_work/hw04_hard.py:
```diff
@@ -4,41 +4,40 @@
 
+def multiply(a, b, c, d, e):
+    r = list(map(int, (a, b, c, d, e)))
+    return r[0] * r[1] * r[2] * r[3] * r[4]
```
здесь лучше организовать через *numbers

rex1k committed:
Lesson 5

Не понял совсем задание хард. Что там делать? Зачем? В коде этом не разобрался совсем. Никаких комментариев, только функция непонятно для чего нужная.

manchenkoff commented on lesson05/home_work/hw05_easy.py:
```diff
@@ -1,10 +1,73 @@
+
+def creator():
+    for x in range(1, 10, 1):
+        f = open(f'dir_{x}.py', 'w')
```
здесь задача создавать директории, а не файлы

manchenkoff commented on lesson05/home_work/hw05_easy.py:
```diff
@@ -1,10 +1,73 @@
+    file_1 = (os.listdir(file))
+    if len(file_1) == 0:
+        print("Directory doesn't have any directories or files")
+    for x in file_1:
```
вместо двух проходов лучше сделать разделение на два списка (в разных ветках условия) и потом сразу их вывести

rex1k committed:
lesson 6

rex1k committed:
les7

fd

rex1k committed:
All done

All done
